iron-fit-behavior.html

```diff
@@ -111,7 +179,40 @@
       return fitTop;
     },
 
+    /**
+     * The element that should be used to position the element when
+     * it opens, if no position target is configured.
+     */
+    get _defaultPositionTarget() {
+      var parent = Polymer.dom(this).parentNode;
+
+      if (parent.nodeType === Node.DOCUMENT_FRAGMENT_NODE) {
+        parent = parent.host;
+      }
+
+      return parent;
+    },
+
+    /**
+     * The horizontal align value, accounting for the RTL/LTR text direction.
+     */
+    get _localeHorizontalAlign() {
+      if (this._isRTL) {
+        // In RTL, "left" becomes "right".
```

Comment:
I agree that `start, end` might be clearer for this use case, but they would be confusing when used together with `no-overlap`.